catalog/spatial/csw/spatial-csw-transformer/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/converter/CswRecordConverter.java

```diff
@@ -306,16 +306,17 @@ public static Serializable convertStringValueToMetacardValue(
      * @return the value that was extracted from {@code reader} and is of the type described by
      * {@code attributeFormat}
      */
-    public static Serializable convertRecordPropertyToMetacardAttribute(String attributeName,
-            HierarchicalStreamReader reader, CswAxisOrder cswAxisOrder) {
-        AttributeDescriptor attributeDescriptor =
-                metacardType.getAttributeDescriptor(attributeName);
-        if (attributeDescriptor != null) {
-            return CswUnmarshallHelper.convertRecordPropertyToMetacardAttribute(attributeDescriptor.getType()
-                    .getAttributeFormat(), reader, cswAxisOrder);
-        }
-        return null;
-    }
+    // TODO - remove?
```

Comment:
Did you want to leave these two TODO removes in here for future attention or delete the code now? If leaving them in, should probably generate a Jira spike to determine whether they can safely be removed.